packages/ui-markdown-editor/src/plugins/withText.js

```diff
@@ -10,8 +11,16 @@ export const withText = (editor) => {
 		if(isBlockHeading(editor)){
 			return;
 		}
-		const currentLine = currentNode.text;
-		matchCases(editor, currentLine);
+
+		const currentLine = currentNode.text
+
+		onkeyup = (ev) => {
```

Comment:
Could this be better handled in our [`onKeyDown`](https://github.com/accordproject/web-components/blob/master/packages/ui-markdown-editor/src/index.js#L95) handler instead of in the `withText` plugin?